### PR TITLE
*: Add a simple bench program

### PR DESCRIPTION
**Pull request by ngaut**

### CODE REVIEW

**Comment by shenli** [comment]
LGTM

**Comment by tiancaiamao** [comment]
did you forget the benchkv/main.go ?

**Comment by ngaut** [comment]
Done.

**Comment by tiancaiamao** [comment]
LGTM

File: benchkv/main.go [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+}
+
+func main() {
+	flag.Parse()
+	log.SetLevelByString("error")
+	Init()
+
+	t := time.Now()
+	batchRW()
+	fmt.Printf("\nelapse:%v\n", time.Since(t))
```
Comment (siddontang) [inline benchkv/main.go]: we can output prom metrics in the end.

Comment (ngaut) [inline benchkv/main.go]: It's not easy to find the bench result when print the metrics.

Comment (siddontang) [inline benchkv/main.go]: You can print metrics first then the bench result.

**Comment by ngaut** [comment]
PTAL @siddontang

**Comment by siddontang** [comment]
LGTM

**Pull request merged**